packages/meld/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
+# This is a template package file for Spack.  We've put "FIXME"
+# next to all the things you'll want to change. Once you've handled
+# them, you can save this file and test your package like this:
+#
+#     spack install meld
+#
+# You can edit this file again by typing:
+#
+#     spack edit meld
+#
+# See the Spack documentation for more information on packaging.
+# ----------------------------------------------------------------------------
+
+from spack.package import *
+from spack.pkg.fnal_art.fnal_github_package import *
+
+
+class Meld(CMakePackage, FnalGithubPackage):
+    """A project for exploring how to meet DUNE's framework needs."""
+
+    repo = "knoepfel/meld.git"
+    version_patterns = []
+
+    maintainers("knoepfel")
+
+    license("Apache-2.0")
+
+    version("develop", branch="main", get_full_repo=True)
+
+    cxxstd_variant("20", "23", default="20", sticky=True)
+
+    depends_on("boost@1.75.0: +json+program_options+stacktrace")
+    depends_on("fmt@:9")
+    depends_on("jsonnet")
+    depends_on("spdlog")
+    depends_on("intel-tbb")
```

Comment:
Please refer to the virtual package instead of the concrete: we usually use `intel-oneapi-tbb`.

Reply:
Unfortunately, I can't use `intel-oneapi-tbb`.  Both `intel-tbb` and `intel-oneapi-tbb` are virtual packages that provide `tbb`.  The difference is that `intel-oneapi-tbb` runs an installation script that cannot be used on macOS Arm systems, whereas `intel-tbb` does support building from source code.

Reply:
@greenc-FNAL, I think I got the sign wrong: `tbb` is the virtual package, and both `intel-oneapi-tbb` and `intel-tbb` are the concrete ones.  So maybe the recommendation is to switch to just `tbb`?

Reply:
I would claim that the inability of `intel-oneapi-tbb` to satisfy the requirement on macOS/arm64 is a packaging/platform issue, and should be managed by an entry in the appropriate `packages.yaml` overriding the virtual package priorities for that platform. The maximum flexibility should be retained in the recipe itself.

Reply:
Thanks for the explanation, Chris.  Recipe adjusted accordingly.